### PR TITLE
[12.0][backport] pos_timeout (from 13.0)

### PR DESCRIPTION
**Pull request by legalsylvain**
trivial backport from 13.0 to 12.0 ported [here](https://github.com/OCA/pos/pull/528) by @acsonefho

The only diff to review is this commit : https://github.com/OCA/pos/pull/568/commits/9e9a5d12f013f7692ea9ac72ea30a1232728f1bb

CC : initial reviewers : @chienandalu, @ivantodorovich, @ThomasBinsfeld

CC : @grap, @quentinDupont. Ref : #/board/144/card/1111

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]
LGTM :+1:
Code review

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by acsonefho — APPROVED** [review]

**Review by quentinDupont — APPROVED** [review]

File: pos_timeout/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Copyright (C) 2017 - Today: Akretion (http://www.akretion.com)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Point of Sale - timeout",
+    "summary": "Set the timeout of the point of sale",
+    "version": "12.0.1.0.1",
+    "category": "Sales/Point Of Sale",
+    "website": "https://odoo-community.org/",
```
Comment (MeetKD) [inline pos_timeout/__manifest__.py]: website -> https://github.com/OCA/pos/

Comment (legalsylvain) [reply pos_timeout/__manifest__.py]: well, in fact it's a backport, so it should be done upstream...

But well, I changed the value.

thanks for your review.

Comment (MeetKD) [reply pos_timeout/__manifest__.py]: :+1:

**Pull request merged**